util/util_test.go

```diff
@@ -61,6 +63,50 @@ var _ = Describe("APID utils", func() {
 		})
 	})
 
+	Context("Forward Proxy Protocol", func() {
```

Comment:
Could we add 1 more test to test the response flow `server response->proxyserver->client`?

Reply:
ok

Reply:
done